src/screens/ChangePasswordScreen/hooks/usePasswordRules.ts

```diff
@@ -0,0 +1,26 @@
+import { RegisterOptions } from 'react-hook-form';
+
+import { ChangePasswordFormInput } from '@src/screens/ChangePasswordScreen';
+
+const usePasswordRules = (
+  validatePassword: () => boolean,
+): {
+  passwordRules: RegisterOptions<ChangePasswordFormInput, 'password'>;
+  passwordConfirmationRules: RegisterOptions<
+    ChangePasswordFormInput,
+    'passwordConfirmation'
+  >;
+} => ({
+  passwordRules: {
+    required: '비밀번호를 입력해주세요',
+    minLength: {
+      value: 8,
+      message: '8글자보다 적습니다',
+    },
+  },
+  passwordConfirmationRules: {
+    validate: validatePassword,
```

Comment:
이런 방식도 좋긴 한데 `getValues` 함수를 파라미터로 받아서 사용해도 좋을거 같아요!
```suggestion
    validate: value => value === getValues('password') || '비밀번호와 값이 다릅니다',
```